octoprint_mrbeam/software_update_information.py

```diff
@@ -89,16 +89,26 @@ def switch_software_channel(plugin, channel):
         plugin.analytics_handler.add_software_channel_switch_event(old_channel, channel)
 
 
-# def _config_octoprint(plugin, tier):
-#     return dict(
-#         octoprint=dict(
-#             checkout_folder="/home/pi/OctoPrint",
-#             pip="https://github.com/mrbeam/OctoPrint/archive/{target_version}.zip",
-#             user="mrbeam",
-#             branch="mrbeam2-stable",
-#             prerelease=(tier == SW_UPDATE_TIER_DEV),
-#         )
-#     )
+def _config_octoprint(plugin, tier):
+    prerelease_channel = None
+    type = "github_release"
+    if tier in [SW_UPDATE_TIER_ALPHA, SW_UPDATE_TIER_BETA]:
+        prerelease_channel = "mrbeam2-{tier}"
+
+    if tier in [SW_UPDATE_TIER_DEV]:
+        type = "github_commit"
```

Comment:
1. Please use `elif`
2. Either change the acceptance criteria in the ticket or change `type` for develop to become prerelease-based and use `develop` as the `prerelease_channel`

Reply:
1. ohh good point
2. I changed the acceptace criteria, I makes more sense to track the github commits for the develop tier